src/lib/ui/thermostat/ClimateSettingStatus.tsx

```diff
@@ -0,0 +1,76 @@
+import type { ClimateSetting } from 'seamapi'
+
+import { ThermostatCoolIcon } from 'lib/icons/ThermostatCool.js'
+import { ThermostatHeatIcon } from 'lib/icons/ThermostatHeat.js'
+import { ThermostatHeatCoolIcon } from 'lib/icons/ThermostatHeatCool.js'
+import { ThermostatOffIcon } from 'lib/icons/ThermostatOff.js'
+
+interface ClimateSettingStatusProps {
+  setting: ClimateSetting
+  iconPlacement?: 'left' | 'right'
+}
+
+export function ClimateSettingStatus({
+  setting,
+  iconPlacement = 'left',
```

Comment:
We can add `temperatureUnit?: 'fahrenheit' | 'celsius'` here

Reply:
https://github.com/seamapi/react/pull/318/commits/6a5367f6509a055bb92d8f82b35d0318004fae15#diff-4641e0e43070de61b08db5514c613b9c7438fd8d64752016e2f3cce89f52abd3R10